docs/reference/shutdown/apis/shutdown-get.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[shutdown-api-get-shutdown]]
+=== Node shutdown status API
+
+Retrieves the status of a node that's in the process of preparing for shutdown,
+indicating whether it is currently ready to shut down or not.
+
+[[shutdown-api-get-request]]
+==== {api-request-title}
+
+`GET _nodes/shutdown`
+
+`GET _nodes/<node-id>/shutdown`
+
+[[shutdown-api-get-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `manage`
+<<privileges-list-cluster,cluster privilege>> to use this API.
+
+* If the <<operator-privileges,{operator-feature}>> is enabled, only operator
+users can use this API.
+
+[[shutdown-api-get-desc]]
+==== {api-description-title}
+
+Retrieves the status of a node that's in the process of preparing to be shut
+down, indicating whether it is in progress, complete, or if it has stalled
+during the process. Detailed statuses of each portion of the shutdown are
+included as well.
+
+[[shutdown-api-get-path-params]]
+==== {api-path-parms-title}
+
+`<node-id>`::
+(Optional, string)
+The ID of a node which is preparing to be shut down. If not provided,
+the statuses of all nodes registered for shutdown will be returned.
+
+[[shutdown-api-get-params]]
+==== {api-query-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+
+[[shutdown-api-get-example]]
+
+This API is only useful after a node has been registered for shutdown:
+
+[source,console]
+--------------------------------------------------
+PUT /_nodes/USpTGYaBSIKbgSUJR2Z9lg/shutdown
+{
+  "type": "restart",
```

Comment:
Minor: I know that it's case-insensitive, but we should probably use all caps (`RESTART`) here in this example just because that's how it gets presented back to the user in the `GET` endpoint

Reply:
I'd leave this as-is. We don't capitalize the strings in other request examples.